### DISCUSSION TITLE
PLG stack (Promtail - Loki - Grafana)

### DISCUSSION THREAD

**Discussion by i5Js** [ricsanfre/pi-cluster#360] [Q&A] [answered]
To complete this stack, I need to deploy Promtail that is used as log collector, right?

**Comment by ricsanfre** [comment, marked as answer]
Yes, If you want the Grafana's full stack. 
In my case, I am using fluentbit/fluentd as common log collector layer for Loki and Elasticsearch. That installation procedure is what you can find in the project documentation. See details in https://picluster.ricsanfre.com/docs/logging-forwarder-aggregator/

**Reply by i5Js** [reply]
Thanks!
